incident_viewer_demo/src/App.js

```diff
@@ -52,31 +81,70 @@ class App extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      hasIncident: false,
+      selected_isp: null, // This is the selected ISP object
     }
 
     // bind handlers
-    this.toggleIncident = this.toggleIncident.bind(this);
+    this.onSelectedClientIspsChange = this.onSelectedClientIspsChange.bind(this);
   }
+
+  onSelectedClientIspsChange(values) {
```

Comment:
What is this variable values referring to? I'm assuming that it is the selected ISPs but let's make this variable name more explicit